docs/Code-Standards.md

```diff
@@ -8,61 +8,78 @@ Please familiarise yourself with our:
-
-- For **containers** or otherwise logic handling code snapshot testing is not desirable and assertion testing is mandatory.
-
-#### Linting standard
-
-A combination of the [Airbnb Styleguide](https://github.com/airbnb/javascript/tree/master/react) and [Prettier](https://github.com/prettier/prettier) is used as our linting standards.
-
-#### Automated Processes
-
-- Linting for code
+- Linting of our code: A combination of the [Airbnb Styleguide](https://github.com/airbnb/javascript/tree/master/react) and [Prettier](https://github.com/prettier/prettier) is used as our linting standards.
+- Issue and PR templates should be followed, then refined by the relevant internal BBC team. We have no templates for hot-fixes which are what we call live and default branch patches; in such cases maximum speed of work is the priority.
+- Most PRs will require code owner approval. This requirement and its enforcement is entirely automated. We have code maintainers across our repos ([Simorgh](https://github.com/bbc/simorgh)/[Psammead](https://github.com/bbc/psammead)), who are responsible for ensuring the quality of the code and are a good source for enquiries.
 - AMP validation for pages
 - BBC a11y validation for pages
 - Lighthouse validation
 - Code Climate Quality tests
 
-#### Manual Processes
+## Manual Processes
 
+- Open your PRs as early as possible, as draft PRs.
+- Add your issues and PRs to the project "Simorgh".
+- If you have paired on a PR you should co-author commits.
 - If latest has a broken build, stop what you are doing and fix it.
-- Code reviews take precedence over implementation
-- Accessibility swarms and UX reviews are required for all front-end components
-
-#### Guideline Exceptions
-
-The coding standards are only guidelines and exceptions can be considered depending on the case.
+- Code reviews take precedence over implementation.
+- Accessibility swarms and UX reviews are required for all front-end components - although they are optional for components with alpha version numbers.
+- Refinement of GitHub issues should be done by the most relevant BBC team.
 
-Examples of exceptions:
+## Merging guidelines and Non-Functional Requirements (NFRs) for release
 
-- Code duplications warnings from Code Climate should be considered on a case-by-case basis (Readability > DRY).
+### Before Merge
 
-- If you have paired on a PR, this can include the person paired with but the reviewer must be a code maintainer
-
-- Alpha components do not require an immediate accessibility swarm or UX review.
+#### Unit Testing:
+- Unit testing is a mixture of Jest snapshot testing and enzyme assertion testing.
+- We push automated testing as far down the testing pyramid as possible. This means that if something can be tested via unit tests, to ensure we have a test coverage of > 80% on this repo.
+- For strictly **presentational** components Snapshots tests must be present and assertion testing is desirable.
```

Comment:
>strictly **presentational** components
Should be in psammead surely?

Reply:
this is intended to be one doc for both repos

Reply:
Will make that clearer in https://github.com/bbc/psammead/issues/1838